Makefile

```
@@ -583,7 +589,15 @@ ifeq ($(detected_OS),Darwin)
 		bin/nim_status_client
 endif
 
+$(NIM_STATUS_LIB): NIM_PARAMS += $(RESOURCES_LAYOUT) -d:useOpenssl3 --noMain --app:staticLib
+$(NIM_STATUS_LIB): $(NIM_SOURCES) | dotherside check-qt-dir $(STATUSGO) $(STATUSKEYCARDGO) $(QRCODEGEN) $(FLEETS) rcc compile-translations deps
```

To compile the static lib it should be enough to have a dependency on the nim sources and the nimbus build system.

```suggestion
$(NIM_STATUS_LIB): $(NIM_SOURCES) | deps-common
```

---

I think we can keep the makefile target simple and call `make status-go` in the top level project instead. In the end, we need these dependencies in the top level cmake, not here.